CamMerrill/Shackelford
# Save artifacts

## Summary
Artifacts will now save. That's it!

## Testing
1) Open up Lab8
2) Dig to reveal some artifacts
3) Move the artifacts
4) Save the game
5) Restart game, notice the artifacts are as you left them!
